addons/account/models/account_move.py

```diff
@@ -2287,11 +2287,34 @@ def is_internal_partner(partner):
         move.message_subscribe(list(all_followers_ids))
         return move
 
-    def post(self):
+    def _post(self, soft=True):
+        """Post/Validate the documents.
+
+        Posting the documents will give it a number, and check that the document is
+        complete (some fields might not be required if not posted but are required
+        otherwise).
+        If the journal is locked with a hash table, it will be impossible to change
+        some fields afterwards.
+
```

Comment:
This new post api looks ugly.

Reply:
that's because the real api is `button_post()`, `_post()` is private and shouldn't be the default usage ?